### PR TITLE
feat(filter) replace select by select2 on host configuration

### PR DESCRIPTION
**Pull request by loiclau**
# Pull Request Template

## Description

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [x] 19.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: www/include/configuration/configObject/host/listHost.php [hunk 1]
```diff
@@ -423,49 +460,13 @@ function setO(_i) {
...
 $tpl->assign('form', $renderer->toArray());
 $tpl->assign('Hosts', _("Name"));
 $tpl->assign('Poller', _("Poller"));
 $tpl->assign('Hostgroup', _("Hostgroup"));
 $tpl->assign('HelpServices', _("Display all Services for this host"));
 $tpl->assign('Template', _("Template"));
 $tpl->assign('Search', _("Search"));
-
-$tpl->assign("StatusFilter", $statusFilter);
+$tpl->assign("StatusFilter", _("StatusFilter"));
```
Comment (kduret) [inline www/include/configuration/configObject/host/listHost.php]: I think you can remove this line

Comment (loiclau) [inline www/include/configuration/configObject/host/listHost.php]:
```suggestion
```

File: www/include/configuration/configObject/host/listHost.ihtml [hunk 2]
```diff
@@ -19,8 +19,8 @@
 			<td>{$form.poller.html}</td>
 			<td>{$form.template.html}</td>
 			<td>{$form.status.html}</td>
-			<td><input type='submit' name='SearchB' value='{$Search}' class="btc bt_success"/></td>
-		</tr>
+			<td>{$form.SearchB.html}</td>
+			</tr>
```
Comment (lgcosta) [inline www/include/configuration/configObject/host/listHost.ihtml]: wrong indentation ?

File: www/include/configuration/configObject/host/listHost.php [hunk 3]
```diff
@@ -226,6 +229,12 @@
 }
 $form->addElement('select2', 'status', "", $statusFilter, $attrHostStatus);
 
+$attrBtnSuccess = array(
+        "class" => "btc bt_success",
+        "onClick" => "window.history.pushState('', '', '?p=" . $p ."');"
```
Comment (lgcosta) [inline www/include/configuration/configObject/host/listHost.php]: 8 spaces to indentation .. is better with 4 here

**Pull request merged**